lib_nn/api/vpu.hpp

```diff
@@ -0,0 +1,57 @@
+#include <cstdint>
+#include <ostream>
+
+#ifndef VPU_HPP_
+#define VPU_HPP_
+
+#include "xs3_vpu.h"
+
+struct vpu_ring_buffer_t
+{
+  int16_t vR[VPU_INT16_EPV];
+  int16_t vD[VPU_INT16_EPV];
+
+#ifdef __cplusplus
+
+  /// Get the 32-bit accumulator with the specified index
+  int32_t get_acc(const int index) const
```

Comment:
Please change this and others to use method naming convention
```suggestion
  int32_t GetAcc(const int index) const
```